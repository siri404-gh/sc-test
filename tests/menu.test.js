import React from 'react';
import { render, unmountComponentAtNode } from 'react-dom';
import { act } from 'react-dom/test-utils';

import Menu from '../src/menu';

describe('Menu', () => {
    let container;

    beforeEach(() => {
        container = document.createElement('section');
        document.body.appendChild(container);
    });

    afterEach(() => {
        unmountComponentAtNode(container);
        container.remove();
        container = null;
    });

    it('Should render', () => {
        act(() => {
            render(<Menu />, container);
        });
        expect(container).toMatchSnapshot();
    });
});