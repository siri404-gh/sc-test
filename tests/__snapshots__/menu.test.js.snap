// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menu Should render 1`] = `
<section>
  <p>
    <a
      href="/"
    >
      Home
    </a>
  </p>
  <p>
    <a
      href="/table"
    >
      Table
    </a>
  </p>
  <p>
    <a
      href="/chart"
    >
      Chart
    </a>
  </p>
</section>
`;
